background.js

```diff
@@ -151,10 +151,15 @@ function getIconForService(service) {
 
 
 function checkNotification(serviceId) {
-	return browser.storage.local.get(serviceId).then((service)=>{
-		var service = service[serviceId];
-		var last = localStorage.getItem('notification/' + service.name + '/last/update');
-		var lastRate = localStorage.getItem('notification/' + service.name + '/last/rate');
+	var keys = [
+		serviceId,
+		'notification/' + serviceId + '/last/update',
```

Comment:
by storing by serviceId instead of by serviceName, we can avoid a second roundtrip. This means all notifications will be cleared when the user updates this extension, but I think that's acceptable if we group it in a release together with other changes.